launcher/downloader/__init__.py

```diff
@@ -11,3 +13,23 @@ def get_handler_for_url(url: str) -> Base:
         'github.com': Github,
         'www.moddb.com': ModDB,
     }.get(host)(url)
+
+
+@retry(stop=stop_after_attempt(3))
+def _download_mod(url: str, download_dir: Path, use_cached: bool = True) -> Path:
+    e = get_handler_for_url(url)
+    file = Path(f"{download_dir}/{e.filename}")
```

Comment:
`file = download_dir / e.filename`

I don't know how Path will react on Windows if you have a `/` in a string during the init.

Reply:
Changed the instantiation.